addons/account/models/account_account.py

```diff
@@ -46,6 +46,7 @@ def _check_account_type_unique_current_year_earning(self):
     company_currency_id = fields.Many2one('res.currency', compute='_compute_company_currency_id')
     company_fiscal_country_code = fields.Char(compute='_compute_company_fiscal_country_code')
     code = fields.Char(string="Code", size=64, tracking=True, compute='_compute_code', search='_search_code', inverse='_inverse_code')
+    _code = fields.Char(company_dependent=True)
```

Comment:
Not a big deal, as we can always do some renaming ourselves once this is merged, but I was just thinking it might be confusing to have a field that starts with `_`.  I'd like to suggest `code_store` for the field name. :smile: 

In any case, the refactoring of `account.code` seems good to me.